examples/hybrid/sphere/balanced_flow_rhoe.jl

```diff
@@ -28,7 +23,7 @@ initial_condition_velocity(local_geometry) =
 
 remaining_cache_values(Y, dt) = merge(
     baroclinic_wave_cache_values(Y, dt),
-    final_adjustments_cache_values(Y, dt; use_rayleigh_sponge = false),
+    final_adjustments_cache_values(Y, dt; use_rayleigh_sponge = sponge),
```

Comment:
I prefer `use_rayleigh_sponge = false` here. If later on we have an option for e.g., viscous sponge, then we could do something like `use_sponge = NoSponge/RayleighDamping/DiffusiveSponge` and make `NoSponge` the default, but for now let's not consider this design in ClimaCore.

Reply:
Oh, I just realize `const sponge = false` is set up above. Then never mind.